src/graphnet/models/graph_builders.py

```diff
@@ -3,8 +3,8 @@
 
```

Comment:
The changes to this file seem to be accidental? Perhaps they should be reverted?

Reply:
I do not understand what this comment

Reply:
I do not understand what this comment refers to @asogaard - Could you help me out?

Reply:
It's just that the only changes to this file seem to be whitespace changes and adding a few imports that are not used? (E.g., `from graphnet.components.pool import group_identical`) If these are indeed unintentional, I was just asking if it would be better to just not modify this file at all in this PR.